### PR TITLE
Add LWIMIQ 2016-10-03

### PR DESCRIPTION
**Pull request by Fryguy**

### CODE REVIEW

**Comment by cybette** [comment]
@Fryguy please add `tags: LWIMIQ` in the frontmatter block so the post will have a category (implemented in the new site). Other than that, LGTM! :+1:

**Comment by JPrause** [comment]
@imtayadeway @himdel @syncrou can you all have a gander. This is for tomorrow's blog entry.

File: source/blog/2016-10-03-last-week-in-manage-iq-swift-vcloud-pg-upgrade-and-euwe-beta.html.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+Along with [ManageIQ/manageiq-appliance-build#165](https://github.com/ManageIQ/manageiq-appliance-build/pull/165)
+and [ManageIQ/manageiq-appliance#91](https://github.com/ManageIQ/manageiq-appliance/pull/91),
+this PR changes the mountpoint location of the PostgreSQL data disk. The
+`pg_upgrade` tool from PostgreSQL has a really cool feature where it will use
+hard links for files that are the same on both sides of the upgrade, which can
+save a significant amount of space. However, the current mountpoint is too deep
+and makes taking advantage of that feature difficult. So, this PR moves us in a
+direction for easier PostgreSQL upgrades.  Thanks
+[Nick](https://github.com/cartbonin)!
```
Comment (syncrou) [inline source/blog/2016-10-03-last-week-in-manage-iq-swift-vcloud-pg-upgrade-and-euwe-beta.html.md]: @Fryguy - `s/cartbonin/carbonin/`

**Comment by imtayadeway** [comment]
@Fryguy looks like in https://github.com/ManageIQ/manageiq.org/pull/353 we went from Mon-Sun to Sun-Sat (and overlapped a Sunday from the week before), continuing into following weeks. I haven't reviewed in the last couple of weeks - was that a conscious decision? Do we want to keep doing that? Personally prefer the former but I'm OK either way

**Comment by imtayadeway** [comment]
:+1: LGTM, just question above about the dates

**Comment by Fryguy** [comment]
I just copied the previous article.  ¯\\\_(ツ)_/¯  Not sure which dates are best

File: source/blog/2016-10-03-last-week-in-manage-iq-swift-vcloud-pg-upgrade-and-euwe-beta.html.md (hunk 1 above)
Comment (Fryguy) [inline source/blog/2016-10-03-last-week-in-manage-iq-swift-vcloud-pg-upgrade-and-euwe-beta.html.md]: Fixed

**Comment by Fryguy** [comment]
Fixed cartbonin and added the tags.

**Comment by JPrause** [comment]
@Fryguy if we're all set,...who will do the merging?

**Pull request merged**